bom/pom.xml

```diff
@@ -52,7 +52,7 @@
         <version.org.jboss.logging.jboss-logging-tools>2.1.0.Final</version.org.jboss.logging.jboss-logging-tools>
         <version.com.fasterxml.classmate>1.3.3</version.com.fasterxml.classmate>
         <version.org.jboss.resteasy>3.0.24.Final</version.org.jboss.resteasy> <!-- Used by CouchDB and Neo4j remote -->
-        <version.com.fasterxml.jackson>2.8.9</version.com.fasterxml.jackson>
+        <version.com.fasterxml.jackson>2.9.7</version.com.fasterxml.jackson>
```

Comment:
The idea here is to be in line with the WF dependencies.

So not sure how we should deal with this.

Reply:
Let me check the one provided by WF.

Reply:
WildFly 13.0.0 uses jackson-core-2.9.5.jar